src/Views/LoginPage/components/Form/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ErrorMessage = ({message}) => {
+
+  return(
+    <div>
+      {message ? <span style={{color : "red"}}>{message}</span> : null}
```

Comment:
use component class name to set style on specific elements. Not inline styling. (Inline styling only allowed in case conditional styling, such as: style={{ display: visible? "block" : "none" })

Reply:
done